### PR TITLE
Feat/autofocus assistant searchbar

### PR DESCRIPTION
**Pull request by JulesBelveze**
## Description

This PR aims at adding autofocus on the assistant search bar when pressing `/`

Issue: https://github.com/dust-tt/tasks/issues/811

## Deploy Plan

- Deploy `sparkle`
- Deploy `front`

### CODE REVIEW

File: sparkle/src/components/Searchbar.tsx [hunk 1]
```diff
@@ -17,78 +17,89 @@ const iconClasses = {
...
-  name: string;
-  size?: "xs" | "sm" | "md";
-  disabled?: boolean;
-  className?: string;
-}) {
-  const clearInputField = () => {
-    onChange?.("");
-  };
+export const Searchbar = forwardRef<
+  HTMLInputElement,
```
Comment (spolu) [inline sparkle/src/components/Searchbar.tsx:21]: Why this type? this is a div at the top? Is it needed?

Comment (spolu) [reply sparkle/src/components/Searchbar.tsx:21]: Why do we need the forwardRef machinery? How is the focus event passed down to the input?

**Review by spolu — COMMENTED** [review]
Noob questions

File: sparkle/src/components/Searchbar.tsx (hunk 1 above)
Comment (JulesBelveze) [reply sparkle/src/components/Searchbar.tsx:21]: I tried without this type and add a bunch of errors telling me to use this type. But curious to hear if there's a better to implement focus 🤓

Comment (JulesBelveze) [reply sparkle/src/components/Searchbar.tsx:21]: @flvndvd any suggestion?

Comment (spolu) [reply sparkle/src/components/Searchbar.tsx:21]: I guess it is typed as the element we're pointing the ref to which makes sense here. Didn't get that we were forwarding the ref duh :p

**Review by spolu — APPROVED** [review]
Looks very legit

File: sparkle/src/components/Searchbar.tsx (hunk 1 above)
Comment (flvndvd) [reply sparkle/src/components/Searchbar.tsx:21]: Yep this is the right way to do it 🙃.

Comment (flvndvd) [reply sparkle/src/components/Searchbar.tsx:21]: Actually, most of our design system should use `forwardRef` so we can have more control from the call sites.

**Pull request merged**